cmd/plume/specs.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 CoreOS, Inc.
+// Copyright 2018 Red Hat Inc.
```

Comment:
None of this code is new, thus the date shouldn't be moved forward.

Reply:
I've migrated this into a new file `types.go`. So, I updated the date forward.

Reply:
When copying existing code, you should always retain the existing copyright header.  The CoreOS acquisition is a bit of a special case, but anyway the old date shouldn't be dropped.  `2016-2018` is okay.

Reply:
Okay